GitUI/GitUI.csproj

```diff
@@ -705,7 +705,7 @@
     <Compile Include="Properties\Resources.Custom.cs">
       <DependentUpon>Resources.resx</DependentUpon>
     </Compile>
-    <Compile Include="RepoObjectsTree\ROT.ContextActions.cs">
+    <Compile Include="LeftPanel\RepoObjectsTree.ContextActions.cs">
```

Comment:
Can it be SidePanel, just in case someone wanted to dock it to the right?:)

Reply:
Agree. What if in future it's docked to the bottom, or floating, or...?

The name should probably represent what is is, not where it is. Like in CSS, semantic labels are more flexible and age better. I'm sure we've all seen something like: `.red-button { background: blue; }` where `cancel-button` would have been a better class name.

Reply:
how about `RepoObjectsPanel` or `BranchTreePanel` though not only branch but also tags and potentially submodules?

Reply:
Didn't you call it `RepoObjectsTree` at some point? No other controls are named with the `Panel` suffix. Others tend to have suffixes like `List` or `Tree`. I think `RepoObjectsTree` seems like a clear and consistent name.